src/zope/app/server/tests/test_accesslog.py

```diff
@@ -19,12 +19,21 @@
 from ZConfig.components.logger import loghandler
 
 try:
+    # ZConfig < 2.9.2
```

Comment:
Should we actually still support this 8 year old version?

Reply:
Probably not, but I was too lazy to remove it.